pkg/virtctl/imageupload/imageupload_test.go

```diff
@@ -31,6 +32,8 @@ import (
 	"kubevirt.io/kubevirt/pkg/virtctl/utils"
 	"kubevirt.io/kubevirt/tests/clientcmd"
 	"kubevirt.io/kubevirt/tests/libstorage"
+
+	"github.com/spf13/cobra"
```

Comment:
Please put this into the block starting at L17 and you could also remove the empty line in L23.

Reply:
Why do you want to have it together with k8s deps? I know that this is the order:
```
go deps
ginkgo deps
k8s deps
kv deps
... any other deps (e.g. third-party - cobra dep)
```